app/lib/tool/test_profile/import_source.dart

```diff
@@ -129,7 +129,7 @@ class _AutoGeneratedImportSource implements ImportSource {
     final hasRepository = hasher('hasRepository', max: 20) > 0;
     final isLegacy = version.contains('legacy');
 
-    final sdkConstraint = isLegacy ? '>=1.12.0 <2.0.0' : '>=2.6.0 <3.0.0';
+    final sdkConstraint = isLegacy ? '>=1.12.0 <2.0.0' : '>=2.13.0 <3.0.0';
```

Comment:
It might be wise to do this in a separate PR. Not that I see anything scary in the golden files.

Reply:
It will change the golden files either way: if we don't change this, the goldens will show the dart3-incompatible badge.